docs/pages/components/index.html

```diff
@@ -9,11 +9,15 @@
 ---
 
 <div class="docs-component-grid">
-    {% for component in site.data.components.items %}
-        <a href="{{component.meta}}.html" class="docs-component-grid__item">
-            <img src="{{site.baseurl}}/images/component-visual/{{component.meta}}.png" alt="{{component.title}}" class="docs-component-grid__img docs-component-grid__img__{{component.meta}}">
-            <br>
-            <h3 class="docs-component-grid--label">{{component.title}}</h3>
-        </a>
-    {% endfor %}
+  {% for component in site.data.components.items %}
```

Comment:
same as above

Reply:
that means that prettierrc file is not used